### PR TITLE
GTA Vice city Cheatsheet

### PR DESCRIPTION
**Pull request by gautamkrishnar**
Continuing from: #1607

-----
IA Page: https://duck.co/ia/view/gta_vice_city_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata post coming soon

File: share/goodie/cheat_sheets/json/gta-vice-city.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "id": "gta_vice_city_cheat_sheet",
+  "name": "Gta Vice City",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: Might as well spell out the full nam:  "Grand Theft Auto: Vice City"

File: share/goodie/cheat_sheets/json/gta-vice-city.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  "aliases": [
+    "vice city",
+  ],
+  "section_order": [
+    "Cheat codes"
+  ],
+  "sections": {
+    "Cheat codes": [
+      {
+        "val": "Aggressive Drivers.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: Let's remove the periods

File: share/goodie/cheat_sheets/json/gta-vice-city.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        "key": "MIAMITRAFFIC"
+      },
+      {
+        "val": "All light weapon.",
+        "key": "THUGTOOLS"
+      },
+      {
+        "val": "All ''heavy'' weapons.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: I would opt for an escaped double quote: `\"`

**Comment by gautamkrishnar** [comment]
done @moollaza :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=GTA Vice City Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/gta_vice_city_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=GTA Vice City Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/gta_vice_city_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/gta-vice-city.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  },
+  "aliases": [
+    "vice city"
+  ],
+  "section_order": [
+    "Cheat codes i",
+    "Cheat codes ii"
+  ],
+  "sections": {
+    "Cheat codes": [
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: @gautamkrishnar mistake in section name

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=GTA Vice City Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/gta_vice_city_cheat_sheet_mobile.png)

**Comment by jagtalon** [comment]
@gautamkrishnar Thanks for removing the XXX cheat codes! I think it might be better to have the column numbers to be "I" and "II" instead of "i" and "ii".

As @moollaza said, there should be a colon in the name, i.e., "Grand Theft Auto: Vice City".

**Comment by gautamkrishnar** [comment]
@jagtalon hehehe :smile: that codes are actually unnecessary :wink: 
Updated with all requested changes :santa:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=GTA Vice City Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/gta_vice_city_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@gautamkrishnar Thanks! This LGTM :+1:

**Comment by moollaza** [comment]
@gautamkrishnar congrats! It's live: https://duckduckgo.com/?q=GTA%20Vice%20City%20Cheat%20Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**